docs/how_to/hypervisors/vagrant_libvirt.md

```diff
@@ -0,0 +1,54 @@
+# Vagrant Libvirt #
+
+This driver enables to trigger tests using libvirtd daemon. It is
+based
+on [libvirt](https://github.com/vagrant-libvirt/vagrant-libvirt)'s
+plugin for vagrant.
+
+## Basic Options ##
+
+With the libvirt daemon on the same host than the running test command
+a simple configuration is
+
+```yaml
+HOSTS
+  centos-puppet-keystone:
+    hostname: puppet-keystone.example.net
+    roles:
+      - master
+    platform: el-7-x86_64
+    box: centos/7
+    box_url:  http://cloud.centos.org/centos/7/vagrant/x86_64/images/CentOS-7.LibVirt.box
+    hypervisor: vagrant_libvirt
+    vagrant_memsize: 4096
+    vagrant_cpus: 2
+```
+
+Those are the usual beaker parameters. Note the `hypervisor`
+parameter. Multiple vms is supported.
+
+
+## Advanced Options and remote libvirt daemon ##
+
+This driver gives you all available parameters from the
+vagrant-libvirt plugin. Beware there could be dragon here as beaker
+has some expectation about the created vms.
```

Comment:
Should be: Beware there could be dragon**s** here as beaker has some expectation**s** about the created VMs.